src/client/datascience/jupyter/kernels/helpers.ts

```diff
@@ -124,12 +124,56 @@ export function getSysInfoReasonHeader(
     }
 }
 
-export function getDisplayNameOrNameOfKernelConnection(
-    kernelConnection: KernelConnectionMetadata | undefined,
-    defaultValue: string = ''
-) {
+export function getDisplayNameOrNameOfKernelConnection(kernelConnection: KernelConnectionMetadata | undefined) {
```

Comment:
We updated the display name for controllers, but I failed to make that change globally.
This will ensure we have the same display name everywhere, in logs, error messages & UI elements.